.travis.yml

```diff
@@ -11,9 +12,8 @@ env:
 services:
   - memcached
 before_install:
-  - sudo apt-get install swig
-  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.3.2.deb && sudo dpkg -i elasticsearch-1.3.2.deb
-  - sudo /usr/share/elasticsearch/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
+  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.3.2.tar.gz && tar xvf elasticsearch-1.3.2.tar.gz
+  - elasticsearch-1.3.2/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
 install:
   - pip install tox
 before_script:
```

Comment:
According to the travis documentation:

```
The caching tars up all the directories listed in the configuration and uploads them to S3, using a secure and protected URL, ensuring security and privacy of the uploaded archives.

Note that this makes our cache not network-local, it's still bound to network bandwidth and DNS resolutions for S3.
```

That's why I didn't bother dealing with a cache (and the cache invalidation issues that may arise?), even more now that we use wheels (which makes it that much faster than compiling locally). I'm not sure the very small hypothetical gain from downloading from S3 instead of pyrepo (or pypi) is worth the trouble. What do you think?